src/modules/Average/index.js

```diff
@@ -0,0 +1,4 @@
+module.exports = [
+    require('./Module'),
+    require('./info.json')
```

Comment:
oh, this is nice! This makes more sense to me because the info.json files are then really still separate and standard. 

Could we adapt the 'all-in-one' format we have above in CONTRIBUTING.md to reference this, somehow? So it's like, "if you're submitting in the browser, there's no `require()` so you have to hard code in the info"?

Reply:
Exactly!!! @jywarren you got this perfectly right, this is the exact idea!!